distributed_filter_envoy/.gitignore

```diff
@@ -0,0 +1,3 @@
+filter.rs
```

Comment:
How do you plan on generating these filters? Via command line?

Reply:
I am thinking you could put the distributed filter into the existing folder, so you do not need to duplicate code.

Reply:
It can also be called filter.rs, that is always generated.

Reply:
Yeah, the thing is the distributed filter and the regular filter don't have a lot of shared code.  They don't share the same filter_base nor do they share the same generated filter functions.  So I feel like that might be a bit confusing to put them in the same folder with the same main file and all.  The filters are generated through command line by adding the -d true flag.

Reply:
Hm okay let's go with this. My worry is you will have double the maintenance effort but ultimately we will have to commit to one design anyway.